blocks/header/headerutil.js

```diff
@@ -298,6 +298,108 @@ const processBondsType = (items) => {
   return processedData;
 };
 
+function loadStockFeed(gaTokenId) {
+  let socket = null;
+  const nifty50 = '4.1!NIFTY 50';
+  const sensex = '1.1!SENSEX';
+
+  // Get current date in IST timezone
+  const now = new Date();
+  const options = {
+    timeZone: 'Asia/Kolkata', hour: '2-digit', minute: '2-digit', weekday: 'long', hour12: false,
+  };
+  const formatter = new Intl.DateTimeFormat([], options);
+  const ISTTime = formatter.format(now);
+
+  // Extract day, time from ISTTime
+  const [day, ...timeParts] = ISTTime.split(' ');
+  const time = timeParts.join(' ');
+
+  // Extract hour, minute, and period from time
+  const [hourMinute] = time.split(' ');
+  const [hour, minute] = hourMinute.split(':').map(Number);
+
+  // Set feedFlag based on the time and day
+  const feedFlag = day !== 'Saturday' && day !== 'Sunday' && ((hour > 9 || (hour === 9 && minute >= 15)) && (hour < 15 || (hour === 15 && minute <= 30)));
+
+  const connect = (hostname, token) => {
+    // eslint-disable-next-line no-undef
```

Comment:
can we remove this linter disabled here?

Reply:
Nope , it needed otherwise lint run fails.

Reply:
trying to understand - how do we know `io` is defined, can it be null, since we are not importing it explicitly?

Reply:
it is coming from the script https://github.com/aemsites/icicidirect/pull/234/files#diff-d02f5299c0b8a5f308bc088a03a095a0785cdd877862859f3bcf9a6a37266db6R11